qlty-check/src/executor.rs

```diff
@@ -245,223 +246,169 @@ impl Executor {
             sub_timer.elapsed().as_secs_f32()
         );
 
+        // Execute config staging operations from the plan
         let sub_timer = Instant::now();
-        let config_file_names = self
-            .plan
-            .invocations
-            .iter()
-            .flat_map(|invocation| &invocation.plugin.config_files)
-            .map(path_to_string)
-            .collect::<std::collections::HashSet<_>>() // Unique
-            .into_iter()
-            .collect::<Vec<_>>();
-
-        let mut repository_config_files = vec![];
-        let walk_builder = self.plan.workspace.walk_builder();
-
-        let mut config_paths = self
-            .plan
-            .invocations
-            .iter()
-            .flat_map(|invocation| invocation.plugin.config_files.clone())
-            .collect::<Vec<PathBuf>>();
-
-        for invocation in self.plan.invocations.iter() {
-            for affects_cache in &invocation.plugin.affects_cache {
-                config_paths.push(PathBuf::from(affects_cache))
-            }
-        }
-
-        let config_globset = config_globset(&config_paths)?;
-
-        for entry in walk_collect_entries_parallel(&walk_builder) {
-            let file_name = entry.file_name().to_str().unwrap();
-            let path = entry.path().to_str().unwrap();
-
-            if config_globset.is_match(file_name)
-                && !path.contains(
-                    self.plan
-                        .workspace
-                        .library()?
-                        .configs_dir()
-                        .to_str()
-                        .unwrap(),
-                )
-            {
-                repository_config_files.push(entry.path().to_owned());
-            }
-        }
-
-        let exported_config_paths = self
-            .plan
-            .invocations
-            .iter()
-            .flat_map(|invocation| invocation.plugin.exported_config_paths.clone())
-            .unique()
-            .collect_vec();
+        let loaded_config_files = self.execute_config_staging_operations()?;
 
         debug!(
-            "Walker found {} config and affects cache files in {:.2}s",
-            repository_config_files.len(),
+            "Staged {} config files in {:.2}s",
+            loaded_config_files.len(),
```

Comment:
We seem to have a collision where sometimes we refer to _"loaded"_ configs and other times we refer to _"staged"_ configs. Would it be appropriate to use "staged" everywhere?